stories/assets/scss/_typography.scss

```diff
@@ -146,4 +146,70 @@ hr {
   border-width: 2px;
 }
 
+// burmese lang
+:lang(my){
+  body {
+    font-family: $Font-Family-NotoSans;
+    line-height: 1.7;
+  }
+
+  h1,h2,h3,h4,h5,h6 {
+    line-height: 1.7;
+  }
+ 
+ h1 {
+    font-size: 2.625rem;
+
+    @include devicebreak(medium) {
+      font-size: 3.875rem;
+    }
+
+    @include devicebreak(large) {
+      font-size: 5rem;
+    }
+  }
+
+  h2 {
+    font-size: 2rem;
+
+    @include devicebreak(medium) {
+      font-size: 2.188rem;
+    }
+
+    @include devicebreak(large) {
+      font-size: 2.813rem;
+    }
+  }
+  h4 {
+  font-size: 1.3rem;
+    @include devicebreak(medium) {
+      font-size: 1.8rem;
+      line-height: 1.7;
```

Comment:
@harishkumar-srijan Already mentioned the line height in 157 line no.

Reply:
done